reversefn.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * This function iterstes through the input string in reverse order and constructs a reversed string by appending each character
+ * 
+ * @param {*} str The input string to be reversed
+ * @returns reversed string
+ */
+function reverseString(str) {
+    let reversed = '';
+    for (let i = str.length - 1; i >= 0; i--) {
+      reversed += str[i];
+    }
+    return reversed;
+  }
+  
+  
+  const originalString = 'Hello, world!';
```

Comment:
The function looks good, and it is well documented. But can we remove the testing part, and just keep the function.

Reply:
Done